CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ Alertmanager API v2 was released in Alertmanager v0.16.0 (released in January
 * [ENHANCEMENT] TSDB: Reload blocks every minute, to detect new blocks and enforce retention more often. #8343
```

Comment:
Not sure how bad of a problem this is, but this PR increases the probability of race conditions with the clean tombstones API.

#8007 changes the logic of this API to reload several times as well, that's when the race was discovered and later on fixed.

Reply:
Was the race present before , but only eevery 2 hours? What is the consequence?

Reply:
Can we have a fix for "just" the race? How hard would that be?

Reply:
Yes, the chance of race was almost impossible before. One would have to call the api exactly when a reload was happening, which was once every 2h.

Now that we reload every minute, if the reload takes too much time, which is unlikely,then we have a chance of race

Reply:
The consequence is the possibility of Prometheus trying to delete a block twice. A block can be cleaned by retention limits and also by the CleanTombstones API. I think it's fine for linux, but on windows that could crash Prometheus

Reply:
There is no race. CleanTombstones() and every minute reloadBlocks() are protected by db.cmtx :)

Reply:
cc @codesome. Do you think I'm right with the comments above?

Reply:
Okey, I saw my mistake now. Sorry for making you folks worry unnecessarily